package.json

```diff
@@ -234,9 +234,11 @@
     "window-size": "^1.1.1"
   },
   "optionalDependencies": {
-    "@cypress/webpack-preprocessor": "^4.1.0",
+    "@cypress/webpack-preprocessor": "^5.1.2",
     "cypress": "^4.0.2",
-    "puppeteer": "^2.1.1"
+    "puppeteer": "^3.0.0",
+    "verdaccio": "^4.5.1",
```

Comment:
Side note `v^4.5.1` is only compatible with Node `v10` or higher.  I see a few lines below the engine node constrains.

Reply:
Storybook only support node v10 and up.